pythran/config.py

```diff
@@ -104,6 +119,11 @@ def lint_cfg(cfgp, **paths):
 
 def make_extension(python, **extra):
 
+    # load platform specific configuration then user configuration
+    cfg = init_cfg('pythran.cfg',
+                   'pythran-{}.cfg'.format(sys.platform),
+                   '.pythranrc',extra.get('config',None))
```

Comment:
this looks better with one line per argument (yes that's neat-picking, but the logic looks good)